cmd/bacalhau/docker_run.go

```diff
@@ -112,7 +117,10 @@ func NewDockerRunOptions() *DockerRunOptions {
 
 func init() { //nolint:gochecknoinits,funlen // Using init in cobra command is idomatic
 	dockerCmd.AddCommand(dockerRunCmd)
-
+	dockerRunCmd.PersistentFlags().BoolVar(
+		&ODR.DryRun, "dry-run", ODR.DryRun,
+		`Run a Job withtout Submitting it to the network`,
```

Comment:
This isn't correct - you're not running a job - you're simulating running a job, right? And outputting what is expected?